libs/user-sdk-event/src/main/java/it/pagopa/selfcare/user/UserUtils.java

```diff
@@ -62,6 +62,7 @@ public static Map<String, String> mapPropsForTrackEvent(TrackEventInput trackEve
         Optional.ofNullable(trackEventInput.getProductId()).ifPresent(value -> propertiesMap.put("productId", value));
         Optional.ofNullable(trackEventInput.getInstitutionId()).ifPresent(value -> propertiesMap.put("institutionId", value));
         Optional.ofNullable(trackEventInput.getProductRole()).ifPresent(value -> propertiesMap.put("productRole", value));
+        Optional.ofNullable(trackEventInput.getGroupMembers()).ifPresent(value -> propertiesMap.put("groupMembers", value.toString()));
```

Comment:
Instead of using value.toString() for groupMembers, consider serializing the set to a more structured format like JSON if the downstream systems expect structured data. For example:
`Optional.ofNullable(trackEventInput.getGroupMembers()).ifPresent(value -> propertiesMap.put("groupMembers", new ObjectMapper().writeValueAsString(value)));
`